SwanProjects/style/base.css

```diff
@@ -0,0 +1,108 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) 2021, SWAN Team.
+|
+| Distributed under the terms of the AGPL License.
+|----------------------------------------------------------------------------*/
+
+.sw-Dialog-content {
+  display: flex;
+  flex-direction: column;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 24px;
+  padding-bottom: 20px;
+  word-wrap: break-word;
+  resize: both;
+}
+.sw-Dialog-content > * + * {
+  margin-top: 20px;
+}
+
+.sw-Dialog-project-icon * {
+  display: flex;
+  width: 39px;
+}
+
+.sw-Dialog-project-name {
+  display: flex;
+  align-items: center;
+}
+
+.sw-Dialog-project-name-input {
+  flex-grow: 1;
+  margin-left: 16px !important;
+}
+
+.sw-Dialog-userscript {
+  display: flex;
+  flex-direction: column;
+  flex: 1 1 auto;
+}
+.sw-Dialog-userscript textarea {
+  font-family: monospace;
+}
+
+.sw-Dialog-userscript-title {
+  display: flex;
+  padding-top: 5px;
+  padding-bottom: 5px;
+}
+.sw-Dialog-userscript-tooltip,
+.sw-Dialog-release-tooltip {
+  padding-left: 5px;
+}
+
+.sw-Dialog-body > .jp-select-wrapper {
+  width: 100%;
+}
+
+/* required to disable "new" notebook and console from the main menu */
+/* with the concept of project, notebook/console can be only created in the launcher, when kernel manager */
+/* have kernels available, otherwise the user will have a kernel error, becuase of kernels are not available*/
```

Comment:
```suggestion
/* Required to disable "new" notebook and console from the main menu. */
/* Notebooks/consoles can be only created from within the launcher of a project, since kernels are only available */
/* inside projects, i.e. kernels are provided by the software environment of a project. */
```